internal/config/features.go

```diff
@@ -19,18 +19,20 @@ package config
 type featureName string
 
 const (
-	FeatureGDS      = featureName("gds")
-	FeatureMOFED    = featureName("mofed")
-	FeatureNVSWITCH = featureName("nvswitch")
-	FeatureGDRCopy  = featureName("gdrcopy")
+	FeatureGDS              = featureName("gds")
+	FeatureMOFED            = featureName("mofed")
+	FeatureNVSWITCH         = featureName("nvswitch")
+	FeatureGDRCopy          = featureName("gdrcopy")
+	FeatureNoAdditionalGIDs = featureName("no-additional-gids")
 )
 
 // features specifies a set of named features.
 type features struct {
-	GDS      *feature `toml:"gds,omitempty"`
-	MOFED    *feature `toml:"mofed,omitempty"`
-	NVSWITCH *feature `toml:"nvswitch,omitempty"`
-	GDRCopy  *feature `toml:"gdrcopy,omitempty"`
+	GDS              *feature `toml:"gds,omitempty"`
+	MOFED            *feature `toml:"mofed,omitempty"`
+	NVSWITCH         *feature `toml:"nvswitch,omitempty"`
+	GDRCopy          *feature `toml:"gdrcopy,omitempty"`
+	NoAdditionalGIDs *feature `toml:"no-additional-gids,omitempty"`
```

Comment:
Why is this reversed (i.e. `no` rather than `allow`) from the flag passed in?

Reply:
Because this has different default behaviour. For the NVIDIA Contianer Runtime I want to generated an internal representation including the `additionalGIDs` container edits for device nodes. This allows users to opt out if there is an issue.

I suppose we could make the argument that if we backport this (which we may want to due to the Jetpack feature request) we should keep the existing behaviour and not make these modifications either.

Reply:
Ok, after having thought this through again, I have disabled this by default and switched the feature flag to `--allow-additional-gids`.